src/pages/taskQueue/index.tsx

```diff
@@ -27,7 +27,7 @@ import TaskQueueTable from "./TaskQueueTable";
 const TaskQueue = () => {
   const taskQueueAnalytics = useTaskQueueAnalytics();
 
-  const { distro } = useParams<{ distro: string }>();
+  const { [slugs.distro]: distro } = useParams();
```

Comment:
do you think we should just make TaskQueue use the `distroId` slug?

Reply:
Good idea we can consolidate them